datalad/distribution/create_sibling.py

```diff
@@ -102,7 +103,7 @@ def _create_dataset_sibling(
         # TODO: Is this condition valid for != '.' only?
         path_exists = True
         try:
-            out, err = ssh(["ls", remoteds_path])
+            out, err = ssh("ls {}".format(sh_quote(remoteds_path)))
```

Comment:
`merging the list into a string, and get into all kind of issue`
yeah -- I had that in mind (merging) and trying to avoid issues by quoting each argument accordingly.  What issues you foresee?